driver/resthandler.go

```diff
@@ -41,12 +41,12 @@ const (
 )
 
 type RestHandler struct {
-	service     *sdk.Service
+	service     *sdk.DeviceService
```

Comment:
The upgrade to latest SDK should be in a separate PR from enabling DependaBot.

Reply:
removed dependabot from this PR